src/views/search_results.hbs

```diff
@@ -61,7 +61,7 @@
 
     </form>
   </div>
-  <div class="fl w-69 mv5 ml3">
+  <div class="w-69 mv5 ml3">
```

Comment:
it needs to be just under the full width so everything would be showing on the page